README.md

```
@@ -1 +1,64 @@
+
+任意:
+
+- 著者 (rich_text)
+- 年 (number)
+- 会議/ジャーナル (rich_text)
+- 被引用数 (number)
+- 分野 (multi_select)
+- ソース (select)
+- Open Access PDF (url)
+- Semantic Scholar ID (rich_text)
+- 要約 (rich_text)
```

markdownlint-cli2 (0.20.0)

[warning] Files should end with a single newline character

(MD047, single-trailing-newline)